modules/ocf_tv/files/i3/config

```diff
@@ -69,14 +74,14 @@ bindsym $mod+Shift+Right move right
 bindsym $mod+semicolon split h
 
 # split in vertical orientation
-bindsym $mod+v split v
+bindsym $mod+slash split v
```

Comment:
What is the reason for making this change?

Reply:
; and v together make literally no sense.